x-pack/plugins/alerting/server/routes/update_rule.ts

```diff
@@ -146,15 +146,13 @@ export const updateRuleRoute = (
       router.handleLegacyErrors(
         verifyAccessAndContext(licenseState, async function (context, req, res) {
           const rulesClient = (await context.alerting).getRulesClient();
-          const actionsClient = (await context.actions).getActionsClient();
+          const { isSystemAction } = (await context.actions).getActionsClient();
```

Comment:
Do we have E2E test cases to assert the correct output for these endpoints?

Reply:
Don't think so but as per Zoom discussion with @XavierM this is out of scope for this PR. Tracked here: https://github.com/elastic/kibana/issues/173761